addons/web/static/lib/daterangepicker/daterangepicker.js

```diff
@@ -942,7 +942,7 @@
                 if (maxDate && time.second(0).isAfter(maxDate))
                     disabled = true;
 
-                if (selected.minute() == i && !disabled) {
+                if (Math.floor(selected.minute() / this.timePickerIncrement) * this.timePickerIncrement === i && !disabled) {
```

Comment:
these math operation should be quick, but maybe `Math.floor(selected.minute() / this.timePickerIncrement) * this.timePickerIncrement` could be computed as a `const` before the loop.

Reply:
Thanks for the review.
Yeah I guess it is possible, but IMHO I don't think it's worth it.
This is not a hot code path and worse case scenario the `timePickerIncrement` is 1, so it will be a loop of 60 iterations which is nothing. In the end it's just a `div` (the floor is quasi free, because it's just taking the quotient part) with a `mul`. Replacing it with an out-of-loop const named let's say `truncatedMinutes` will force the next reader to scroll up just to see what is the formula, instead of having it inlined.
Also I am pretty sure the JIT that will be executing this code will surely have some Loop-invariant code motion optimization enabled. 
But all of this is just my opinion, if the JS team says they think it is necessary, yeah sure I will hoist the formula out of the loop.